internal/pkg/template/workload.go

```diff
@@ -611,6 +611,7 @@ func withSvcParsingFuncs() ParseOption {
 			"hasSecrets":           hasSecrets,
 			"fmtSlice":             FmtSliceFunc,
 			"quoteSlice":           QuoteSliceFunc,
+			"quote":                QuoteStringFunc,
```

Comment:
nit: we can remove `QuoteStringFunc`:
```suggestion
			"quote":                strconv.Quote,
```